core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -77,11 +87,19 @@ oppia.factory('CollectionNodeObjectFactory', [
     // frontend exploration summary tile displaying. Changes to the returned
     // object are not reflected in this domain object.
     CollectionNode.prototype.getExplorationSummaryObject = function() {
-      // TODO(bhenning): This should be represented by a frontend summary tile
-      // domain object that is also shared with the exploration editor/viewer.
+      // TODO(bhenning): This should be represented by a frontend summary domain
+      // object that is also shared with the exploration editor/viewer.
       return angular.copy(this._explorationSummaryObject);
     };
 
+    // Sets the raw exploration summary object stored within this node.
+    CollectionNode.prototype.setExplorationSummaryObject = function(
+        explorationSummaryBackendObject) {
+      // TODO(bhenning): This function should be removed once the exploration
+      // summary object is represented by a frontend summary domain object.
```

Comment:
At first I didn't understand this TODO, since the frontend summary domain object is indeed an exploration summary object. Maybe s/exploration summary object/exploration backend summary object.